docs/install-azure-pipelines.md

```diff
@@ -288,6 +288,10 @@ jobs:
             # [Optional, default: "FALSE"]
             # Displays all existing vulnerabilities, including the ones that were added by the pull request.
             # JF_INCLUDE_ALL_VULNERABILITIES: "TRUE"
+
+            # [Optional, default: "FALSE"]
+            # Keep old comments that were added by previous Frogbot scans on the pull request.
```

Comment:
Keep old comments that were added by previous Frogbot scans on the pull request.
-->
When adding new comments on pull requests, keep old comments that were added by previous scans.